web/index.html

```diff
@@ -1463,19 +1474,24 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 	},
 
 	'disk.iotime': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of milliseconds spend by all operation. Read from fields 4 and 8.'
```

Comment:
The sum of the duration (in milliseconds) of all completed I/O operations. This number can exceed the interval if the disk is able to execute I/O operations in parallel.